docs/source/workbench/instrumentviewer.rst

```diff
@@ -70,65 +67,208 @@ Pick Tab
+
+.. |peakComparison| image:: ../images/PickTabCPCompare.png
+
+.. image:: ../images/Workbench/InstrumentViewer/PeakComparisonExample.png
+
+Peak Alignment Tool
+###################
+
+The |peakAlign| tool allows for the calculation of in-plane and out-of-plane angles between a user defined plane and an out-of-plane peak in Q-space. The plane
+must be defined first, by selecting two peaks. Clicking while they are within the hatched box will select them, and mark them with a blue circle. Subsequent peaks
+will be highlighted by green circles. The information box will contain the in-plane angles (phi) and out-of-plane angles (theta) for each green peak.
+
+Selections can be cleared by right-clicking the instrument view and selecting ``Clear alignment plane``.
+
+.. |peakAlign| image:: ../images/PickTabCPAlign.png
+
+.. image:: ../images/Workbench/InstrumentViewer/PeaksAlignmentExample.png
 
 
 .. _instrumentviewer_draw_tab:
 
 Draw Tab
 --------
-The Draw tab contains tools for creating and editing geometrical shapes which can be used for selecting regions of interest (ROI), masking or grouping detectors.
-The tab contains a mini toolbar, a shape property browser and a set of buttons to use the shapes.
+
+.. |ellipse| image:: ../images/PickTabEllipseButton.png
+.. |rectangle| image:: ../images/PickTabRectButton.png
+.. |ellipseRing| image:: ../images/PickTabElRingButton.png
+.. |rectangleRing| image:: ../images/PickTabRectRingButton.png
+.. |sector| image:: ../images/PickTabSectorButton.png
+.. |arbitrary| image:: ../images/PickTabArbitraryShape.png
+
+The Draw tab contains tools for creating and editing geometrical shapes, which can be used for selecting regions of interest (ROI) or for masking or grouping detectors.
+The tab contains a smaller toolbar, a shape property browser, and a set of buttons to create shapes.
 
 .. figure:: ../images/Workbench/InstrumentViewer/DrawTab.png
     :align: center
     :width: 635
 
-More doc will be migrated from MantidPlot as new features being consolidated.
+Masks are created by using six tools:
+
+- |ellipse| Ellipse
+- |rectangle| Rectangle
+- |ellipseRing| Ellipse Ring
+- |rectangleRing| Rectangle Ring
+- |sector| Sector
+- |arbitrary| Brush: This allows for drawing of masks arbitrarily. The size of the brush can be changed by scrolling.
+
+There is no limit on the number of shapes. The shapes can be selected and edited by selecting |edit|.
+Click on the shaded part of a shape and drag to move it around or move a control point to resize.
+
+.. |edit| image:: ../images/PickTabEditButton.png
+
+.. image:: ../images/Workbench/InstrumentViewer/DrawShapesExample.jpg
+
+The property browser displays the parameters of the selected shape. All sizes are given in the coordinates of the unwrapped surface the shapes are drawn on.
+For example, for a spherical surface it will be polar and azimuth angles in radians. The shapes are "attached" to the surface and move with it when the view is zoomed or moved.
```

Comment:
```suggestion
For example, for a spherical surface it will be polar and azimuth angles in radians. The shapes are 'attached' to the surface and move with it when the view is zoomed or moved.
```

Reply:
Why does this need to be changed?

Reply:
I think single quotes for turn of phrase as discussed above.

Reply:
Is that a standard? I'd use double quotes for that kind of thing like I'd do with my fingers when speaking

Reply:
I guess its a personal preference thing so lets leave it as it is.